person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unkonwn', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    (@age >= 18)
+  end
+
+  public
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] Please make sure that you're calling the right method here. You've named the method `of_age?` but when you called you, you typed `is_of_age?`